systemtest/src/test/java/io/strimzi/systemtest/upgrade/AbstractKRaftUpgradeST.java

```diff
@@ -179,6 +186,60 @@ protected void doKafkaConnectAndKafkaConnectorUpgradeOrDowngradeProcedure(
         verifyProcedure(testStorage.getNamespaceName(), upgradeDowngradeData, testStorage.getContinuousProducerName(), testStorage.getContinuousConsumerName());
     }
 
+    /**
+     * Verifies that the Kafka Connector FileSink is receiving messages as expected.
+     *
+     * @param testStorage Test-related configuration and storage
+     */
+    private void verifyKafkaConnectorFileSink(final TestStorage testStorage) {
+        String connectorPodName = kubeClient().listPods(
+            testStorage.getNamespaceName(),
+            Collections.singletonMap(Labels.STRIMZI_KIND_LABEL, KafkaConnect.RESOURCE_KIND)
+        ).get(0).getMetadata().getName();
+
+        KafkaConnectUtils.waitForMessagesInKafkaConnectFileSink(
+            testStorage.getNamespaceName(),
+            connectorPodName,
+            DEFAULT_SINK_FILE_PATH,
+            testStorage.getMessageCount()
+        );
+    }
+
+    /**
+     * Waits for the Kafka cluster and Kafka Connect to roll if the target version is supported.
+     *
+     * @param testStorage        Test-related configuration and storage
+     * @param upgradeKafkaVersion Kafka version details
+     */
+    private void maybeWaitForRollingUpdate(final TestStorage testStorage,
```

Comment:
You are using just Namespace name here, so you don't need to pass whole TestStorage

Reply:
updated. Thanks